pkg/ingester/instance.go

```diff
@@ -53,6 +53,11 @@ var (
 		Name:      "ingester_memory_streams",
 		Help:      "The total number of streams in memory per tenant.",
 	}, []string{"tenant"})
+	memoryStreamsLabels = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
`memoryStreamsLabelsBytes` would be a better name for the variable. The current name suggests the count of the labels to me.

Reply:
good point 👍 Done.